clang/test/CIR/CodeGen/static-local.cpp

```diff
@@ -0,0 +1,78 @@
+// RUN: %clang_cc1 -triple x86_64-unknown-linux-gnu -fclangir -emit-cir %s -o %t.cir
+// RUN: FileCheck --input-file=%t.cir %s --check-prefix=CIR
+// RUN: %clang_cc1 -triple aarch64-none-linux-android21 -fclangir -emit-cir -clangir-disable-passes %s -o %t.cir
+// RUN: FileCheck --input-file=%t.cir %s --check-prefix=CIRGEN
+// RUN: %clang_cc1 -triple x86_64-unknown-linux-gnu -fclangir -emit-llvm %s -o %t.cir
+// RUN: FileCheck --input-file=%t.cir %s --check-prefix=LLVM
```

Comment:
Can you update the testcase to reflect what you can currently emit? It's fine if you want to leave the other things around for now, just add TODOs or FIXMEs.

Reply:
Looks like you already did, nevermind!

Reply:
Actually, I got confused because lowering prepare changes were so big I they got folded by github. Reinstating my point: please remove the lowering prepare changes from this PR and change the testcase to only cover CIRGen. I think there's more we can move to CIRGen that don't require all those interfaces (I asked for the threadsafe attribute in the other review but somehow you've missed addressing that)

Reply:
After ditching interfaces and leaving lowering prepare for another PR, this mostly looks good to me (with the other minor changes requested)